app/src/main/java/org/listenbrainz/android/ui/screens/feed/FeedScreen.kt

```diff
@@ -450,67 +495,68 @@ private fun MyFeed(
 
         item { StartingSpacer() }
 
-        items(count = pagingData.itemCount) { index: Int ->
-
-            pagingData[index]?.apply {
-                AnimatedVisibility(
-                    visible = uiState.isDeletedMap[event.id] != true,
-                    enter = expandVertically(),
-                    exit = shrinkVertically()
-                ) {
-                    eventType.Content(
-                        event = event,
-                        parentUser = parentUser,
-                        isHidden = uiState.isHiddenMap[event.id] == true,
-                        onDeleteOrHide = {
-                            onDeleteOrHide(
-                                event,
-                                eventType,
-                                parentUser
-                            )
-                        },
-                        dropDownState = dropdownItemIndex.value,
-                        index = index,
-                        onDropdownClick = {
-                            dropdownItemIndex.value = if (dropdownItemIndex.value == null) {
-                                index
-                            } else {
-                                null
-                            }
-                        },
-                        onRecommend = {
-                            recommendTrack(event)
-                            dropdownItemIndex.value = null
-                        },
-                        onPersonallyRecommend = {
-                            personallyRecommendTrack(index)
-                            dropdownItemIndex.value = null
-                        },
-                        onReview = {
-                            review(index)
-                            dropdownItemIndex.value = null
-                        },
-                        onPin = {
-                            pin(index)
-                            dropdownItemIndex.value = null
-                        },
-                        onOpenInMusicBrainz = {
-                            uriHandler.openUri("https://musicbrainz.org/recording/${event.metadata.trackMetadata?.mbidMapping?.recordingMbid ?: return@Content}")
-                            dropdownItemIndex.value = null
-                        },
-                        onClick = {
-                            onPlay(event)
-                            dropdownItemIndex.value = null
-                        },
-                        goToUserPage = goToUserPage,
-                        goToArtistPage = goToArtistPage
-                    )
+        if (inRefreshingState) {
+            items(4) {
```

Comment:
Can we add more items here, such that we are able to fill up the whole screen? In order to fill the whole screen, you can measure the layout using subcompose layout and then use BoxWithConstraints (outside Lazy column) to get max available height.